src/hooks/useCombinedWallets.ts

```diff
@@ -60,7 +60,8 @@ export const useCombinedWallets = () => {
       (connector) =>
         isWalletInstalled(connector.id) &&
         // We should not show already connected connectors
-        account.connector?.id !== connector.id,
+        account.connector?.id !== connector.id &&
+        connector.id !== 'safe',
```

Comment:
is there a way to grap the actual safe connector id from the wallet management package instead of hardcoding the string here?

Reply:
I am not sure, I have checked and we are looking for the id/name whenever we are trying to identify multisig in general (like the example below)

<img width="678" alt="image" src="https://github.com/lifinance/jumper.exchange/assets/45823872/7203762c-7c2f-4de0-a578-facdedc6c73b">